lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -518,6 +519,55 @@ class DecomposeAtenMatmulOp : public OpRewritePattern<AtenMatmulOp> {
 };
 } // namespace
 
+// ReLU6(x) = min(max(0, x), 6) = min(Relu(x), 6)
+static Value getRelu6Results(PatternRewriter &rewriter, Location loc,
+                             Value input) {
+  Type inputType = input.getType();
+  Value none = rewriter.create<Torch::ConstantNoneOp>(loc);
+  Value relu = rewriter.create<AtenReluOp>(loc, inputType, input);
+  Value onesTensor = rewriter.create<AtenOnesLikeOp>(
+      loc, inputType, input, /*dtype=*/none, /*layout=*/none, /*device=*/none,
+      /*pin_memory=*/none, /*memory_format=*/none);
+  Value constantSix =
+      rewriter.create<Torch::ConstantIntOp>(loc, rewriter.getI64IntegerAttr(6));
+  Value sixTensor =
```

Comment:
you can use `PseudoAtenFillScalarOp` now

Reply:
Done.